stock_available_unreserved/models/quant.py

```diff
@@ -17,6 +17,9 @@ class StockQuant(models.Model):
     @api.depends("product_id", "location_id", "quantity", "reserved_quantity")
     def _compute_contains_unreserved(self):
         for record in self:
+            if isinstance(record.id, models.NewId):
```

Comment:
Do it for all records as since V13, records don't have default value before compute call.

Reply:
I don't understand. Then, what condition could be used?

Reply:
Mmh, I see (didn't look further code).

But, if you read _get_available_quantity code, you see that rounding value comes from product_id.uom_id.rounding.

So, maybe uom_id is not defined or rounding == 0.00. The condition would be on these element (testing NewId is more a hack).

Reply:
1. uom_id is a required field from products.
2. rounding have a SQL constraint that don't allow you to set it to = 0.

The only case this function fails is when the compute has through by a NewId record.

I would like to find a better way to fix this :)

Reply:
Can you add a comment in code explaining the reason for this? Our future us will be grateful :wink: 

If nobody has a better solution we can merge like this now.

Reply:
Done!